addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -846,7 +846,13 @@ MockServer.include({
      * @private
      */
     _mockMailChannelExecuteCommand(args) {
-        const [ids, commandName] = args.args;
+        let ids, commandName;
+        if (args.kwargs && args.kwargs.command) {
+            ids = args.args;
```

Comment:
Isn't `ids` always `args.args[0]`?
And `commandName` is `args.kwargs.command || args.args[1]`

Reply:
Does not seem to be something like that, once the command is passed as kwargs the other as args and once id is a list and the other one it's an int. Both seems to work.

![image](https://user-images.githubusercontent.com/48204996/95888258-248b8e00-0d81-11eb-8542-c9f6e98b710a.png)

Reply:
The one on the right is the server being smart and `browse` accepting more than the expected list of ids (int, newid or even string).

I would prefer if we changed it to always send a list of id, so we don't need to handle that in the mock server.